papaparse.js

```diff
@@ -1468,15 +1469,46 @@
 				'You need to set Papa.SCRIPT_PATH manually.'
 			);
 		var workerUrl = Papa.SCRIPT_PATH || AUTO_SCRIPT_PATH;
-		// Append 'papaworker' to the search string to tell papaparse that this is our worker.
-		workerUrl += (workerUrl.indexOf('?') !== -1 ? '&' : '?') + 'papaworker';
+
+		if(Papa.SCRIPT_PATH_AS_BLOB_URL){
+			workerUrl = createDummyBlobUrlFromUrl(workerUrl);
+		}
+		else{			
+			// Append 'papaworker' to the search string to tell papaparse that this is our worker.
+			workerUrl += (workerUrl.indexOf('?') !== -1 ? '&' : '?') + 'papaworker';
+		}
+
 		var w = new global.Worker(workerUrl);
 		w.onmessage = mainThreadReceivedMessage;
 		w.id = workerIdCounter++;
 		workers[w.id] = w;
 		return w;
 	}
 
+	function createDummyBlobUrlFromUrl(script_path) {
```

Comment:
This function should remove the suffix added to know that's a worker and then load the correct javascript file.

Reply:
Forget about this comment.